packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -425,37 +114,6 @@ QUnit.test('the {{this}} helper should not fail on removal', function() {
   equal(view.$().text(), '');
 });
 
-QUnit.test('should update the block when object passed to #unless helper changes', function() {
```

Comment:
This is a dup of https://github.com/emberjs/ember.js/pull/12914/files#r52058901 as far as I can tell?